config/initializers/101-lograge.rb

```diff
@@ -101,18 +101,25 @@
           end
         end
 
-      if ENV["LOGSTASH_URI"]
+      if ENV["ENABLE_LOGSTASH_LOGGER"] == "1"
         config.lograge.formatter = Lograge::Formatters::Logstash.new
 
         require "discourse_logstash_logger"
 
         config.lograge.logger =
-          DiscourseLogstashLogger.logger(uri: ENV["LOGSTASH_URI"], type: :rails)
+          DiscourseLogstashLogger.logger(
+            logdev: Rails.root.join("log", "#{Rails.env}.log"),
+            type: :rails,
+            customize_event:
+              lambda do |event|
+                event["database"] = RailsMultisite::ConnectionManagement.current_db
+              end,
+          )
 
         # Remove ActiveSupport::Logger from the chain and replace with Lograge's
-        # logger
-        Rails.logger.chained.pop
-        Rails.logger.chain(config.lograge.logger)
+        # logger.
+        Logster.logger.instance_variable_get(:@chained).delete(Logster.logger.chained.first) # Reach into instance variable but this will be irrelevant in Rails 7.1
```

Comment:
In Rails 7.1, the `ActiveSupport::Logger` will no longer be chained to Logster.

Reply:
https://github.com/discourse/logster/pull/238/files#r1660938639